src/main/kotlin/com/buildit/bookit/auth/OpenIdAuthenticator.kt

```diff
@@ -13,8 +13,17 @@ interface JwtAuthenticator {
 }
 
 class BookitSecurityContext(private val request: HttpServletRequest) : SecurityContext {
-    fun allowFakeTokens(props: BookitProperties) =
-        props.allowTestTokens ?: listOf("localhost", "integration").any { request.serverName.startsWith(it) }
+    private val log = LoggerFactory.getLogger(this::class.java)
+    fun allowFakeTokens(props: BookitProperties): Boolean {
+        val isNotProdRequest = listOf("localhost", "integration").any { request.serverName.startsWith(it) }
+        log.info("Is non-prod request: $isNotProdRequest")
```

Comment:
Would like to have used `log.debug`, but couldn't get those to appear in output even after fooling with the logging properties in `application.yml`.